lib/crud_responder/locales/en.yml

```diff
@@ -0,0 +1,12 @@
+en:
```

Comment:
Usual place for locales is `config/locales` and there are few already. No need to create a new one